extensions/wikia/WAM/controllers/api/WAMApiController.class.php

```diff
@@ -136,10 +136,10 @@ public function getWAMLanguages() {
 		$wamDay = $this->request->getVal( 'wam_day', null );
 		$wamDates = $this->getMinMaxWamIndexDateInternal();
 
-		if ( empty( $wamDay ) ) {
+		if ( empty( $wamDay ) || $wamDay > $wamDates[ 'max_date' ] ) {
```

Comment:
I would say yes it's possible.  Testing may be tricky.  The date ranges in the Service/Controller/Model are frequently manipulated with + or -1 or 2 for various "local" reasons.  There are a lot of code paths so it's difficult to know where all the points of usage are.  The underlying code adds +1 here, for instance, so that's why it has to subtract 2.  😦  But I never figured out why that +1 is even there.

https://github.com/Wikia/app/blob/dev/includes/wikia/services/WAMService.class.php#L195